amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -93,40 +113,66 @@ describe('SearchPage', () => {
       expect(props.getPopularTables).toHaveBeenCalled();
     });
 
-    it('calls getSelectedTabByResourceType with correct value', () => {
-      expect(getSelectedTabByResourceTypeSpy).toHaveBeenCalledWith('table');
-    });
-
-    it('calls setState with result of getSelectedTabByResourceType', () => {
-      expect(setStateSpy).toHaveBeenCalledWith({ selectedTab:  mockResourceType });
+    it('calls setState', () => {
+      expect(setStateSpy).toHaveBeenCalledWith({ selectedTab: ResourceType.table });
```

Comment:
checking `expect(setStateSpy).toHaveBeenCalledWith({ selectedTab: mockSanitizedUrlParams.currentTab });` more accurately tests the logic of this method.